openstudiocore/src/osversion/VersionTranslator.hpp

```diff
@@ -162,6 +162,7 @@ class OSVERSION_API VersionTranslator {
   int m_nObjectsFinalIdf;
   int m_nObjectsFinalModel;
   bool m_isComponent;
+  std::vector<IdfObject> m_cbeccSizingObjects;
```

Comment:
maybe tag with a version number in case this happens again?  m_cbeccSizingObjects_190?